src/node/rpc/test/frontend_test_infra.h

```diff
@@ -139,7 +139,7 @@ auto get_vote(
 {
   const auto getter = create_request(
     nullptr,
-    fmt::format("proposals/{}/votes/{}", proposal_id, voter),
```

Comment:
That sounds unused.

Reply:
Yes - there's several functions in `frontend_test_infra.h` that are only used by `member_voting_test`, so can be removed with it.

Reply:
Ok, I'll have a look for them.